src/modules/submodules/consensus.py

```diff
@@ -70,7 +71,7 @@ def _get_member_info(self, blockstamp: BlockStamp) -> MemberInfo:
 
         # Defaults for dry mode
         current_ref_slot, deadline_slot = self._get_current_frame(blockstamp)
-        is_member, last_report_ref_slot, member_report_for_current_ref_slot = True, 0, b''
+        is_member, last_member_report_ref_slot, member_report_for_current_ref_slot = True, 0, b''
```

Comment:
It changes nothing

Reply:
in dry mode `last_report_ref_slot` var was not used in any way further. I think it should be initial value for `last_member_report_ref_slot` to init `MemberInfo`